### DISCUSSION TITLE
GraphQL Repo Vulnerabilities

### DISCUSSION THREAD

**Discussion by hinorahere** [community/community#24411] [Apps, API and Webhooks]
<p>I need to retrieve the severity of my repo’s security vulnerabilities for each individual project using Graphql.</p>
<p>I have seen similar questions asked here with several answers saying it is only available to check through Github’s UI. I was wondering if anyone had a good workaround or if I am doing something incorrectly?</p>
<p>Basically something like this:</p>
<p>query {<br>
  repositoryOwner(login: “repoOwner”) {<br>
    login<br>
    repositories(last:10) {<br>
      edges{<br>
      node {<br>
        name<br>
        securityVulnerabilities(first:3) {</p>
<p>          nodes {<br>
            severity<br>
          }  </p>
<p>       }<br>
    }<br>
}<br>
}<br>
}<br>
}</p>

**Comment by wei** [comment]
<p>Please look at <a href="https://developer.github.com/v4/object/repositoryvulnerabilityalert/" rel="nofollow noopener">https://developer.github.com/v4/object/repositoryvulnerabilityalert/</a>. It is still in preview though.</p>
